cmd/notation/logout.go

```diff
@@ -16,8 +16,9 @@ func logoutCommand(opts *logoutOpts) *cobra.Command {
 		opts = &logoutOpts{}
 	}
 	return &cobra.Command{
-		Use:   "logout [server]",
-		Short: "Log out the specified registry hostname",
```

Comment:
not: `Short` seems to be longer than `Long`.

Reply:
Updated: both short and long description use `Log out from the logged in registries`